tools/biotransformer/.shed.yml

```diff
@@ -0,0 +1,10 @@
+categories:
+- Metabolomics
+description: BioTransformer is a tool for prediction of small molecule metabolism in mammals.
+homepage_url: https://gitlab.ics.muni.cz/recetox/mass-spectrometry/biotransformer
+long_description: |
+  This tool is a complete wrapper for biotransformer.
```

Comment:
It is useful to provide a paragraph of what the tool does on what type of data to help future users. We could take it from the tool's `<help>` section.

Reply:
Long description is updated to give some meaningful info to the users.